### ISSUE TITLE
How low is the loss value?

### ISSUE THREAD

**Issue by 2000ZRL** [hynann/MoLingo#4]
Dear authors,

Thank you for releasing your code. I am trying to reimplement your work. Could you please tell me what value the flow matching loss will decrease to?

Thanks for your time.

**Comment by hynann** [comment]
Hi, thanks for your interest. The loss for our 272-dim MoLingo model training converges at around 0.22-0.26. You can also try our released training scripts.
